### ISSUE TITLE
[expo-media-library] It only crashed in Standalone App.

### ISSUE THREAD

**Issue by Woong25** [expo/expo#10168]
### Summary of Issue 
Crash when used in standalone apps on Google Play Store.
But when I run it as a virtual machine (expo start) it doesn't crash.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
```Expo CLI 3.27.4 environment info:

Expo CLI 3.27.4 environment info:
    System:
      OS: Windows 10 10.0.18362
    Binaries:
      Node: 12.16.2 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.14.4 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6514223
    npmPackages:
      expo: 38.0.10 => 38.0.10
      react: 16.11.0 => 16.11.0
      react-dom: 16.11.0 => 16.11.0
      react-native: https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz => 0.62.2
      react-native-web: ~0.11.7 => 0.11.7
      react-navigation: ^4.4.0 => 4.4.0
      expo-media-library": "9.2.1
    Expo Workflow: managed
```

### Reproducible Demo
```
const getFolderList = async () => {
        const albums = await MediaLibrary.getAlbumsAsync({
            mediaType: [MediaLibrary.MediaType.photo],
        })
        setFolderList(albums);
};

const getMediaData = async (first=30, after, album, folder, type) => {
        const data = await MediaLibrary.getAssetsAsync({
            first,
            after,
            album,
            sortBy: [[ MediaLibrary.SortBy.modificationTime, false ], [ MediaLibrary.SortBy.creationTime, false ]],
            mediaType: [MediaLibrary.MediaType.photo],
        });
}

useEffect(() => {
        getMediaData();
        getFolderList();
}, [])
```

### Stack Trace
```
java.lang.RuntimeException: 
  at android.os.AsyncTask$4.done (AsyncTask.java:399)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:383)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:252)
  at java.util.concurrent.FutureTask.run (FutureTask.java:271)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:919)
Caused by: java.lang.IllegalArgumentException: 
  at android.database.DatabaseUtils.readExceptionFromParcel (DatabaseUtils.java:170)
  at android.database.DatabaseUtils.readExceptionFromParcel (DatabaseUtils.java:140)
  at android.content.ContentProviderProxy.query (ContentProviderProxy.java:423)
  at android.content.ContentResolver.query (ContentResolver.java:951)
  at android.content.ContentResolver.query (ContentResolver.java:887)
  at android.content.ContentResolver.query (ContentResolver.java:843)
  at expo.modules.medialibrary.GetAlbums.doInBackground (GetAlbums.java:34)
  at expo.modules.medialibrary.GetAlbums.doInBackground (GetAlbums.java:18)
  at android.os.AsyncTask$3.call (AsyncTask.java:378)
  at java.util.concurrent.FutureTask.run (FutureTask.java:266)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:919)
```
 
![image](https://user-images.githubusercontent.com/44948010/93011392-db54ec80-f5d0-11ea-8bbb-f81a99cc7162.png)

I updated the versions of expo and expo-media-library but it still crashes.

Sorry. It was difficult to prepare a reproducible demo because it only crashes in standalone apps.

Thanks!

**Comment by dipamsen** [comment]
I am having this same problem

**Comment by ahmetbicer** [comment]
Hi guys, i was having the same problems and open an issue #9839 a month ago. Please use expo-media-library 9.2.0 you can install it via `npm install expo-media-library@next`. It will probably fix your issue.

**Comment by AdamJNavarro** [comment]
Hey @Woong25  and @dipamsen,

Can you let me know what version(s) of `expo-media-library` you tested that caused the crash? I see in your diagnostics log it says you're running `9.2.1`.

Cheers,
Adam

**Comment by Woong25** [comment]
@AdamJNavarro 
Hello
The tested version is `9.2.1`.
(There is version information for `expo-media-library` in the Bug Report Environment Group.)

Please reply if you need more information.
Thank you.

**Comment by nebalceroc** [comment]
Same problem here.

Using version 8.2.1

**Comment by barthap** [comment]
`getAlbumsAsync()` was broken on Android. This is fixed in #9666. Fix will be released soon, with SDK 39.
